server/graphql-schema.js

```diff
@@ -492,6 +492,12 @@ const graphqlSchema = gql`
         A human-readable version of the command, such as "Control+Alt+Down"
         """
         text: String!
+        """
+        The AT mode this command may be getting ran in, such as "quickNavOn".
+        The same command can be ran during the same test, but in a different
+        mode
+        """
+        settings: String
```

Comment:
It is possible to make this an enum?

Reply:
The values here won't be static. If the `support.json` is updated with a new AT + settings (or just a new setting in general for a currently imported AT), and the auto-importer triggers, it will fail to query the relevant content if this is an enum of the `settings` values we know of now.

Reply:
What do you think about calling this "atOperatingMode"? I have a couple of motivations for the suggestion. One is that since the word "settings" is plural it makes me suspicious that I'm dealing with something like a comma separated list. Another is that settings is quite a broad term, and are not obviously dealing specifically with the AT, for example it could be referring to operating system settings, browser settings, keyboard settings like holding the shift key, any kind of settings, and I think that's one of the things which is confusing me. I suggested "operatingMode" because I feel like "atMode" is a bit too restrictive a term.

Also, can you improve the documentation for this field? I think we should exhaustively list out all the values which are currently used, one by one. Also if you have a link to a specification listing all possible options, putting that here would be hugely helpful as well.

Reply:
**On settings > atOperatingMode request**

> I suggested "[atO]peratingMode" because I feel like "atMode" is a bit too restrictive a term.

Agreed, I can make that change.

> ... since the word "settings" is plural it makes me suspicious that I'm dealing with something like a comma separated list. 

Note that eventually, [this may happen](https://github.com/w3c/aria-at/issues/1002). That's based on the spec request of [AT_KEY-commands.csv > settings](https://github.com/w3c/aria-at/wiki/Test-Format-Definition-V2#settings).

**On documentation update request**

> Also, can you improve the documentation for this field? I think we should exhaustively list out all the values which are currently used, one by one.

Listing them out will require a change [any time they're updated](https://github.com/w3c/aria-at/commit/399b44ece13542345425c6593227d76dfd9631d2), which would be of an unknown frequency. That would make that bit of inline documentation irrelevant for a bit of time and I'm doubtful of how best that will be kept updated.

> Also if you have a link to a specification listing all possible options, putting that here would be hugely helpful as well.

I don't but let's suggest updating the [v2 Test Format Definition](https://github.com/w3c/aria-at/wiki/Test-Format-Definition-V2) with a section on that and link there. Otherwise, the only spec for that is to check out [each key in ats[].settings in `support.json`](https://github.com/w3c/aria-at/blob/master/tests/support.json), which isn't too easy.